Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM python:alpine
 
 
 # metadata
-ENTRYPOINT ["/radicale/docker/entrypoint.sh"]
+ENTRYPOINT ["/sbin/tini", "--", "/radicale/docker/entrypoint.sh"]
```

Comment:
As far as I understand, this is not necessary. All started processes are running in foreground, so the signals will get forwarded, too.

Reply:
thanks for the hint. that's a leftover of several combinations i tested, also with `su-exec` which would be neat to use in the entrypoint script but unfortunately spawned an extra process in this context.